deploy/melange.tmpl.yaml

```diff
@@ -0,0 +1,21 @@
+package:
+  name: embedded-cluster-operator
+  version: ${VERSION}
+  epoch: 0
+  description: Embedded Cluster Operator
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    repositories:
+      - https://packages.wolfi.dev/os
+    keyring:
+      - https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
+    packages:
+      - busybox
+
+pipeline:
+  - runs: |
+      set -ex
+      cp manager "${{targets.destdir}}/manager"
```

Comment:
why not build the binary here? it's more portable that way and seems like the established pattern

Reply:
i agree. ill fix this